en/operators.md

```diff
@@ -4,6 +4,8 @@ language: 'en'
 version: '0.12'
 ---
 # Operators
+This chapter is about Zephir's operators. To read about operator pecedence refer to chapter [Operator Precedence](/{{ page.version }}/{{ page.language }}/operator-precedence).
```

Comment:
This article describes Zephir's operators. For precedence you can check the [Operator Precedence](/{{ page.version }}/{{ page.language }}/operator-precedence) article.